iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveTableUtil.java

```diff
@@ -125,7 +125,7 @@ private static List<DataFile> getDataFiles(RemoteIterator<LocatedFileStatus> fil
     while (fileStatusIterator.hasNext()) {
       LocatedFileStatus fileStatus = fileStatusIterator.next();
       String fileName = fileStatus.getPath().getName();
-      if (fileName.startsWith(".") || fileName.startsWith("_")) {
+      if (fileName.startsWith(".") || fileName.startsWith("_") || fileName.endsWith("metadata.json")) {
```

Comment:
So metadata files are handled, but are snapshot files (that are avro formatted) going to cause similar issues?

Reply:
There will be no snapshot files at this point since the data files are not yet imported. We only have the first metadata file that belongs to an empty iceberg table.

Reply:
Got it, thanks!